internal/pkg/overlay/overlay.go

```diff
@@ -350,8 +352,12 @@ func RenderTemplateFile(fileName string, data TemplateStruct) (
 	err error) {
 	backupFile = true
 	writeFile = true
-	tmpl, err := template.New(path.Base(fileName)).Option("missingkey=default").Funcs(template.FuncMap{
-		// TODO: Fix for missingkey=zero
+
+	// Build our FuncMap
+	funcMap := template.FuncMap{
+		"tr": func(source, old, new string) string {
+			return strings.Replace(source, old, new, -1)
+		},
```

Comment:
Although I like the `tr` on the command line, I would remove it from the templates and rely on the sprig functions only. This closes the doors for requests to add special functions because they are useful for this special use case.